example.env.create-cluster

```diff
@@ -0,0 +1,8 @@
+export AWS_PROFILE=moj-cp
```

Comment:
Is this strictly required? For the cluster creation bits we should have the appropriate profiles specified in the terraform provider configuration blocks.

Reply:
I think we need it for kops anyway - I can't see any way to supply the aws profile to the kops command. If we need it for that, we may as well use it for terraform too.